tc/core/cpu/cpu_tc_executor.cc

```diff
@@ -60,8 +60,7 @@ ProfilingInfo CpuTcExecutor::profileUnchecked(
     const std::vector<const void*>& inputs,
     const std::vector<void*>& outputs) const {
   LOG(ERROR) << "NYI: CpuTcExecutor::profileUnchecked";
-  return ProfilingInfo{
-      Duration(std::chrono::microseconds(static_cast<int64_t>(999999999999))),
-      Duration(std::chrono::microseconds(static_cast<int64_t>(999999999999)))};
+  return ProfilingInfo{Duration(Duration::fromMicroSeconds(999999999999)),
```

Comment:
are we sure this value is representable?  why not `Duration::max()` ?

Reply:
I really dislike those sentinel values. We should be using optional (and we need to agree if we are going to roll our own or use a 3rd party one).

Reply:
This is _not_ an optional return value.  CPU run is not implemented, so this just returns something to silence the warning

Reply:
I also think `Duration::max()` would be more appropriate here,
or just use the default constructor.